openlibrary/macros/EditionNavBar.html

```diff
@@ -32,7 +32,7 @@
     </li>
   </ul>
   $if not desktop_only:
-    <span class="nav-arrow">
-      <img src="/static/images/nav-arrow.svg" role="img">
-    </span>
+    <button class="nav-arrow" title="$_('Go to next section')">
+      <img src="/static/images/nav-arrow.svg" alt="">
+    </button>
```

Comment:
I didn't notice this earlier, but this drastically affected the style of the navigation arrow:
![image](https://github.com/user-attachments/assets/cb19d09a-40b5-42fb-b433-3211cfde32b6)
_`nav-arrow` now has some `<button>` styles_

Can you add `background: none;` and `border: none;` to the `nav-arrow` class, [here](https://github.com/internetarchive/openlibrary/blob/4bace241f47722c5bb8768a0dafb41a178753581/static/css/components/nav-bar.less#L59-L61)?